packages/astro/src/core/errors/errors-data.ts

```diff
@@ -595,7 +595,7 @@ export const ExpectedImage = {
 	name: 'ExpectedImage',
 	title: 'Expected src to be an image.',
 	message: (src: string, typeofOptions: string, fullOptions: string) =>
-		`Expected \`src\` property for \`getImage\` or \`<Image />\` to be either an ESM imported image or a string with the path of a remote image. Received \`${src}\` (type: \`${typeofOptions}\`).\n\nFull serialized options received: \`${fullOptions}\`.`,
+		`Expected \`src\` property for \`getImage\` or \`\<Image /\>\` to be either an ESM imported image or a string with the path of a remote image. Received \`${src}\` (type: \`${typeofOptions}\`).\n\nFull serialized options received: \`${fullOptions}\`.`,
```

Comment:
Please also adjust the `\n\n` as required for displaying the error message properly! This is currently rendering as-is when pulled into docs.